lib/memfs/io.rb

```diff
@@ -18,6 +18,20 @@ def read(path, *args)
       ensure
         file.close if file
       end
+
+      def write(path, string, offset = 0, open_args = nil)
+        open_args ||= [File::WRONLY, encoding: nil]
+
+        if offset > 0
```

Comment:
This line would fail if offset is `nil` and the value is valid. Even if offsets are not implemented there should be no error except when a user actually tries to use it :)